src/sentry/pipeline/__init__.py

```diff
@@ -226,6 +226,12 @@ def current_step(self):
 
     def error(self, message):
         context = {"error": message}
+        extra = {
+            "organization_id": self.organization.id if self.organization else None,
+            "provider": self.provider.key,
```

Comment:
We're guaranteed to have `self.provider`, right?

Reply:
Yes